### PR TITLE
virustotal-cli 0.8.0

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/virustotal-cli.rb [hunk 1]
```diff
@@ -11,21 +11,13 @@ class VirustotalCli < Formula
...
-      system "dep", "ensure", "-vendor-only"
-      system "go", "build", "-ldflags",
-             "-X #{repo}/cmd.Version=#{version}",
-             "-o", bin/"vt", "./vt/main.go"
-      prefix.install_metafiles
-    end
+    system "go", "build", "-ldflags",
+            "-X cmd.Version=#{version}",
+            "-o", bin/"vt", "./vt/main.go"
+    prefix.install_metafiles
```
Comment (dawidd6) [inline Formula/virustotal-cli.rb:20]: Could be removed too.

Comment (chenrui333) [reply Formula/virustotal-cli.rb:20]: I thought these metafiles are for license and such?

Comment (dawidd6) [reply Formula/virustotal-cli.rb:20]: It's called automatically.

Comment (chenrui333) [reply Formula/virustotal-cli.rb]: I see, just removed it.

**Review by dawidd6 — APPROVED** [review]

**Pull request closed without merging**